modules/Microsoft.DigitalTwins/digitalTwinsInstances/.test/common/deploy.test.bicep

```diff
@@ -0,0 +1,120 @@
+targetScope = 'subscription'
+
+// ========== //
+// Parameters //
+// ========== //
+
+@description('Optional. The name of the resource group to deploy for testing purposes.')
+@maxLength(90)
+param resourceGroupName string = 'ms.digitaltwins.digitaltwinsinstances-${serviceShort}-rg'
+
+@description('Optional. The location to deploy resources to.')
+param location string = deployment().location
+
+@description('Optional. A short identifier for the kind of deployment. Should be kept short to not run into resource-name length-constraints.')
+param serviceShort string = 'dtdticom'
+
+@description('Optional. Enable telemetry via a Globally Unique Identifier (GUID).')
+param enableDefaultTelemetry bool = true
+
+// ============ //
+// Dependencies //
+// ============ //
+
+// General resources
+// =================
+resource resourceGroup 'Microsoft.Resources/resourceGroups@2021-04-01' = {
+  name: resourceGroupName
+  location: location
+}
+
+module nestedDependencies 'dependencies.bicep' = {
+  scope: resourceGroup
+  name: '${uniqueString(deployment().name, location)}-nestedDependencies'
+  params: {
+    virtualNetworkName: 'dep-<<namePrefix>>-vnet-${serviceShort}'
+    managedIdentityName: 'dep-<<namePrefix>>-msi-${serviceShort}'
+    eventHubName: 'dt-${uniqueString(serviceShort)}-evh-01'
+    eventHubNamespaceName: 'dt-${uniqueString(serviceShort)}-evhns-01'
+    serviceBusName: 'dt-${uniqueString(serviceShort)}-sb-01'
+    serviceBusTopicName: 'dt-${uniqueString(serviceShort)}-sbtp-01'
```

Comment:
Question: Do the diverse topic names need to be unique? If not, we could also just hardcode them in the dependencies file like we do for example for the virtual network subnet - effectively reducing the amount of parameters

Reply:
Correct it doesn't require a unique name. I will hard code the value into dependency template.

Reply:
Renamed